### PR TITLE
feat: update Athena saved queries based on new table

### PR DESCRIPTION
**Pull request by llmin**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

update Athena saved queries based on new tables including event, event_parameter and user.

## Implementation highlights

(describe how the merge request does for feature changes, share the RFC link if it has)

## Test checklist

- [ ] add new test cases
- [x] all code changes are covered by unit tests
- [ ] end-to-end tests
  - [ ] deploy web console with CloudFront + S3 + API gateway
  - [ ] deploy web console within VPC
  - [ ] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [ ] with S3 sink
  - [ ] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [x] Athena
  - [ ] deploy with reporting

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

## Miscellaneous

- [ ] introduce new symbol link source file(s) to be shared among infra code, web console frontend, and web console backend

### CODE REVIEW

File: src/analytics/private/athena-saved-queries.ts [hunk 1]
```diff
@@ -16,16 +16,15 @@ import { CfnNamedQuery } from 'aws-cdk-lib/aws-athena';
...
+import { AthenaAnalyticsStackProps } from '../parameter';
 
-export interface AthenaSavedQueryProps {
-  readonly databaseName: string;
-  readonly workGroup: string;
-  readonly eventTable: string;
-}
+export type AthenaSavedQueryProps = AthenaAnalyticsStackProps;
 
 export type MustacheParamType = {
```
Comment (zxkane) [inline src/analytics/private/athena-saved-queries.ts:23]: You might define via typescript's Omit/Partial/Pick from `AthenaSavedQueryProps`.

File: src/analytics/private/athena-saved-queries.ts [hunk 2]
```diff
@@ -34,17 +33,19 @@ export class AthenaSavedQuery extends Construct {
     super(scope, id);
 
     const mustacheParam: MustacheParamType = {
```
Comment (zxkane) [inline src/analytics/private/athena-saved-queries.ts:35]: how about cast the props as `MustacheParamType`

File: src/analytics/private/athena-saved-queries.ts (hunk 1 above)
Comment (llmin) [reply src/analytics/private/athena-saved-queries.ts]: fixed

File: src/analytics/private/athena-saved-queries.ts (hunk 2 above)
Comment (llmin) [reply src/analytics/private/athena-saved-queries.ts]: fixed

**Review by zxkane — APPROVED** [review]

**Pull request merged**